app/helpers/sms_helper.rb

```diff
@@ -25,6 +25,12 @@ def format_sms_messages_field(sms, field)
       recips
     when "from" then
       user_with_phone(sms.sender, sms.from)
+    when "body" then
+      content_tag(:span) do
+        "".html_safe <<
+          sms.body <<
+          content_tag(:div, "#{I18n.t('sms.error')}: #{sms.error_message}", class: "error-msg")
```

Comment:
TERRIFIC. Nailed it!

Reply:
Oop! One small thing -- you mustn't add the error message if there is no error, right!?

Reply:
Hmm, and I just learned something new. Rubocop prefers this:

```
out = []
out << content_tag(:li, "one")
out << content_tag(:li, "two")
safe_join(out)
```

So perhaps let's switch to that. Then we don't need `html_safe` at all!

Reply:
do you think it would make sense to have a default value of "" in the db? @smoyth